doc-content/enterprise-only/release-notes/rn-731_fixed-issues-ref.adoc

```diff
@@ -0,0 +1,85 @@
+[id='rn-731-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.3.1
+
+{PRODUCT} 7.3.1 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+
+== {CENTRAL}
+
+* The `NoClassDefFoundError` appears in the server log file when a service task is installed in a project [https://issues.jboss.org/browse/RHPAM-1947[RHPAM-1947]]
+* You cannot create a project in an offline environment [https://issues.jboss.org/browse/RHPAM-1282[RHPAM-1282]]
+* {CENTRAL} cannot connect to a co-located controller when deployed at the same time [https://issues.jboss.org/browse/RHPAM-1920[RHPAM-1920]]
+* The package name white list should contain the projects default package [https://issues.jboss.org/browse/RHPAM-1890[RHPAM-1890]]
+* Column sorting is not retained after filters are applied in the Task in box [https://issues.jboss.org/browse/RHPAM-1700[RHPAM-1700]]
+
+== Form modeler
+* When you start a process instance in {CENTRAL} the modal window contains the previously rendered form until the new form renders [https://issues.jboss.org/browse/RHPAM-1951[RHPAM-1951]]
+* The multiple selector throws an error when edited [https://issues.jboss.org/browse/RHPAM-1861[RHPAM-1861]]
+
+== Installer
+* The default placeholder for the controller URL should contain `business-central` when installing {KIE_SERVER} and {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1774[RHPAM-1774]]
+* The controller URL is not updated in XML files when only {KIE_SERVER} is being installed [https://issues.jboss.org/browse/RHPAM-1781[RHPAM-1781]]
+* You cannot use the installer to install {PRODUCT} on JWS 5.1 [https://issues.jboss.org/browse/RHPAM-2038[RHPAM-2038]]
+
+== OpenShift
+
+* Setting the Maven mirror URL does not exclude {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1994[RHPAM-1994]]
+* The KIE server image does not run correctly whe  EJB is initialized first [https://issues.jboss.org/browse/RHPAM-1970[RHPAM-1970]]
+* {CENTRAL} throws an exception when the user clicks a deployment unit [https://issues.jboss.org/browse/RHPAM-1952[RHPAM-1952]]
+* `ServerTemplateConverter` breaks with `KieServerMode` mixed-case [https://issues.jboss.org/browse/RHPAM-1975[RHPAM-1975]]
+
+== Process designer
+
+* You cannot open a callable process that contains spaces and extended chars [https://issues.jboss.org/browse/RHPAM-2006[RHPAM-2006]]
+
+
+== Process engine
+
+* {KIE_SERVER} is not removed from the controller view if the server is killed or crashes [https://issues.jboss.org/browse/RHPAM-1654[RHPAM-1654]]
+* The newly added `org.jbpm.correlationkey.length` system property does not truncate the *Value* column in the `CorrelationPropertyInfo` table [https://issues.jboss.org/browse/RHPAM-1961[RHPAM-1961]]
+* The {KIE_SERVER} client does not reuse the marshaller for JMS [https://issues.jboss.org/browse/RHPAM-1988[RHPAM-1988]]
+* An EJB timer error occurs in development mode for an existing process during deploy [https://issues.jboss.org/browse/RHPAM-1985[RHPAM-1985]]
+
+== PMML
```

Comment:
I think we might as well just delete this. This has absolutely no impact on users.

Reply:
Removed